src/aks-preview/HISTORY.md

```diff
@@ -12,6 +12,7 @@ Release History
 
 0.4.52
 +++++
+* Add "--enable-azure-rbac" for enabling Azure RBAC for Kubernetes authorization
```

Comment:
you may need to add a new version in HISTORY.md and https://github.com/Azure/azure-cli-extensions/blob/master/src/aks-preview/setup.py.

Reply:
added